src/app/home/HomeComponent.jsx

```diff
@@ -1,9 +1,11 @@
 import React, { Fragment } from 'react';
 import '../../style/home.scss';
+import Navbar from '../navBar/NavbarComponent.jsx';
 
 const HomeComponent = () => {
   return (
     <Fragment>
+      <Navbar />
```

Comment:
Let us add this to `App.js`, the same way the footer is there.

Reply:
@habibaudu

Reply:
ok